frontend/express/public/javascripts/countly/countly.vue.components.js

```diff
@@ -372,4 +373,2886 @@
+                    isOpened: function(newState) {
+                        if (!newState) {
+                            this.reset();
+                        }
+                    }
+                },
+                mounted: function() {
+                    this.stepContents = this.$children.filter(function(child) {
+                        return child.isContent && child.role === "default";
+                    });
+                    this.sidecarContents = this.$children.filter(function(child) {
+                        return child.isContent && child.role === "sidecar";
+                    });
+                    this.isMounted = true;
+                },
+                methods: {
+                    tryClosing: function() {
+                        this.$emit("close", this.name);
+                    },
+                    copyOfEdited: function() {
+                        var copied = JSON.parse(JSON.stringify(this.initialEditedObject));
+                        return this.beforeObjectCopy(copied);
+                    },
+                    setStep: function(newIndex) {
+                        if (newIndex >= 0 && newIndex < this.stepContents.length) {
+                            this.currentStepIndex = newIndex;
+                        }
+                    },
+                    prevStep: function() {
+                        this.setStep(this.currentStepIndex - 1);
+                    },
+                    nextStep: function() {
+                        this.beforeLeavingStep();
+                        if (this.isCurrentStepValid) {
+                            this.setStep(this.currentStepIndex + 1);
+                        }
+                    },
+                    reset: function() {
+                        this.$v.$reset();
+                        this.resetLocalState();
+                        this.setStep(0);
+                    },
+                    submit: function() {
+                        this.beforeLeavingStep();
+                        if (!this.$v.$invalid) {
+                            this.$emit("submit", this.beforeSubmit(JSON.parse(JSON.stringify(this.editedObject))));
+                            this.tryClosing();
+                        }
+                    },
+                    afterObjectCopy: function(newState) {
+                        return newState;
+                    },
+                    beforeObjectCopy: function(newState) {
+                        return newState;
+                    },
+                    beforeSubmit: function(editedObject) {
+                        return editedObject;
+                    },
+                    getInitialLocalState: function() {
+                        return {};
+                    },
+                    resetLocalState: function() {
+                        this.localState = this.getInitialLocalState();
+                    },
+                    beforeLeavingStep: function() { }
+                },
```

Comment:
Codacy found an issue: [Multiline support is limited to browsers supporting ES5 only.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6339769)